docs/user/building-images.md

```diff
@@ -201,6 +201,9 @@ sudo podman manifest push --all \
     docker://${OCI_IMAGE_REPO}-unified:${OCI_IMAGE_TAG}
 ```
 
+> [!NOTE]
+If your device relies on an OS image from a private repository, authentication credentials (pull secrets) must be placed in the appropriate system path `/etc/ostree/auth.json`. Authentication must exist on the device before it can be consumed.
```

Comment:
Note looks good, I think it belongs further up (e.g. line 112), though, because this section is about the raw disk images.

Reply:
updated and added a ref to rhel docs as per our conv.